core/Pos/Core/Configuration.hs

```diff
@@ -129,7 +126,6 @@ withCoreConfigurations conf@CoreConfiguration{..} confDir mSystemStart mSeed act
 
         theSystemStart <- case mSystemStart of
             Just it -> do
-                logInfo $ sformat ("withConfiguration using custom system start time "%shown) it
```

Comment:
Why did you delete it?

Reply:
Eliminates the `WithLogger` constraint.

Reply:
What's the problem with `WithLogger` constraint? Just use `usingLoggerName "name"` to eliminate it at the calling site.
Or at least you can change it to `liftIO $ usingLoggerName "withCoreConfigurations" $ logInfo …` here and eliminate `WithLogger` constraint this way.